client/src/ui/PageFooter.tsx

```diff
@@ -26,43 +50,37 @@ export const PageFooter = () => {
           flexWrap: "nowrap",
         }}
       >
-        <Link
-          href="#"
-          style={{ margin: "0 40px", color: "#000", textDecoration: "none" }}
-        >
-          <Typography>Explore</Typography>
-        </Link>
-        <Link
-          href="#"
-          style={{ margin: "0 40px", color: "#000", textDecoration: "none" }}
-        >
-          <Typography>About Us</Typography>
-        </Link>
-        <Link
-          href="#"
-          style={{ margin: "0 40px", color: "#000", textDecoration: "none" }}
-        >
-          <Typography>Contact</Typography>
-        </Link>
+        {["Explore", "About Us", "Contact"].map((text, index) => (
```

Comment:
won't these be links? It should also have a href, not just the name. Additionally, don't hardcode it like this. Place it in a variable like the images array